### PR TITLE
Add setting to change size of Arabic pane font size

### LINKED ISSUE #372: Arabic pane font size needs to be adjusted separately to primary pane
At the moment users can choose a font size and it will be applied equally to all edition panes. However appearance settings in Nammu need to include a separate line for Arabic pane font size when arabic mode is on. 

We should also probably reorder the way the options are being shown. It is not very intuitive at the moment to have as an option the font size of the console.

### PR DESCRIPTION
**Pull request by giordano**
Fix #372.

In one of the commits I also rationalised the functions that build the menu.

### CODE REVIEW

File: python/nammu/view/AtfAreaView.py [hunk 1]
```diff
@@ -250,10 +250,13 @@ def refresh(self):
         doc.setCharacterAttributes(0, doc.getLength() + 1, attrs, False)
 
         # Also apply new fontsize to arabic pane
+        arabic_attrs = self.controller.arabic_area.getInputAttributes()
```
Comment (raquel-ucl) [inline python/nammu/view/AtfAreaView.py]:
```python
for area_name, style in [['edit_area',  'edit_area_style'],['arabic_area', 'arabic_pane_style']]:
     area = getattr(self.controller, area_name)
     font = set_font(config[style]['fontsize']['user'])
     if area_name == 'edit_area':
        self.controller.syntax_highlighter.font = font
        self.controller.syntax_highlighter.setup_attribs()
     attrs = area.getInputAttributes()
     StyleConstants.setFontSize(attrs, font.getSize())
     doc = area.getStyledDocument()
     doc.setCharacterAttributes(0, doc.getLength() + 1, attrs, False)

```

Add some comments where it makes sense!

File: python/nammu/view/EditSettingsView.py [hunk 2]
```diff
@@ -151,100 +153,100 @@ def build_working_dir_panel(self, constraints, panel):
...
-                                           insets=Insets(10, 10, 80, 10))
-        panel.add(server_label, constraints)
-
         self.combo = self.build_servers_combobox()
-        constraints = self.add_constraints(constraints, weightx=0.70,
-                                           gridx=1, gridy=1, gridwidth=2,
-                                           fill=GridBagConstraints.HORIZONTAL)
-        panel.add(self.combo, constraints)
+        self.build_drop_down_menu(constraints, panel, self.combo,
+                                  JLabel("ORACC server location:"), 1,
```
Comment (raquel-ucl) [inline python/nammu/view/EditSettingsView.py]: Replace all "ORACC" mentions with "Oracc".

File: resources/config/settings.yaml [hunk 3]
```diff
@@ -124,4 +124,9 @@ edit_area_style:
         default: 14
         user: 14
 
+arabic_pane_style:
```
Comment (raquel-ucl) [inline resources/config/settings.yaml]: Rename to `arabic_area_style` so naming is consistent.

File: resources/config/settings.yaml [hunk 4]
```diff
@@ -124,4 +124,9 @@ edit_area_style:
...
+    fontsize:
+        default: 14
```
Comment (raquel-ucl) [inline resources/config/settings.yaml]: I'd make the arabic font default to 16-18 because it's always too small compared to the non arabic font.

**Comment by giordano** [comment]
I think I addressed all comments in this PR as well.

**Pull request merged**